api/src/main/java/io/hyperfoil/api/config/Phase.java

```diff
@@ -32,11 +32,9 @@ public final class Phase implements Serializable {
    public final Model model;
    public final boolean isWarmup;
    public final Map<String, SLA[]> customSlas;
+   public final StartWithDelay startWithDelay;
```

Comment:
After some reasoning I decided to implement this as a dedicated class, as it basically includes two different things:
- a phase ref as a string
- a delay
I decided this because in this way the phase ref will have eactly the same format used for all others `startAfter*` properties, so I could reused some of the methods to checks cyclic deps and so on.

If we don't want to have a complex object here, we could:
- split this in two separated fields, don't like as these two fields are correlated.
- create a single string as a result, similarly to what has been done for phase refs, e.g., `<phase-ref>:<delay>`